data/json/professions.json

```diff
@@ -5138,5 +5138,145 @@
                 "panties"
             ]
         }
+    },
+    {
+        "type": "profession",
+        "ident": "escape_con",
+        "name": "Escaped Prisioner",
+        "description": "When the cataclysm started you took your chance, now you roam looking for a place to stay. Hopefully those days workin in prison pay off..",
+        "points": -2,
+        "skills": [
+            {
+                "level": 1,
+                "name": "mechanics"
+            },
+            {
+                "level": 3,
+                "name": "fabrication"
+            }
+        ],
+        "items": {
+            "both": [
+                "technician_shirt_gray",
+                "striped_pants",
+                "greatcoat",
+                "manual_knives",
+                "knife_steak",
+                "gloves_fingerless"
+            ],
+            "male": [
+                "boxer_shorts"
+            ],
+            "female": [
+                "bra",
+                "panties"
+            ]
+        }
+    },
+    {
+        "type": "profession",
+        "ident": "bad_burglar",
+        "name": "Amateur Burglar",
+        "description": "You thought life'd be better after the catclysm. Oh, were you wrong.",
+        "points": 2,
+        "items": {
+            "both": [
+                "socks",
+                "boots",
+                "pants",
+                "tshirt",
+                "hoodie",
+                "gloves_light",
+                "balclava",
+                "swag_bag",
+                "makeshift_crowbar",
+                "picklocks"
```

Comment:
That's a very mediocre gear composition. Most of this can be created from scratch in the wild. This makes the class very redundant. I can't think of a justification or suggestions to keep it in core, just dump it in More Professions or remove it altogether.

Reply:
K.